### PR TITLE
Fix/importing meshes

### PR DESCRIPTION
**Pull request by JorxSS**
Fixed bug:

* Now all the meshes have the current world position, not just 0,0,0.
* Also even if the mesh is called equal all of them are at their respective position

### CODE REVIEW

**Review by yalania — CHANGES_REQUESTED** [review]
I would like to have a look again to the skeleton importer, I changed it too fast and want to make sure it works for all models

File: Engine/ResourceManagement/Importer/ModelImporter.cpp [hunk 1]
```diff
@@ -114,6 +117,9 @@ FileData ModelImporter::ExtractData(Path& assets_file_path, const Metafile& meta
 	}
 	aiReleaseImport(scene);
 
+	std::string config_to_serialize;
+	model.GetSerializedString(config_to_serialize);
+	App->filesystem->Save("/Hola.txt", config_to_serialize);
```
Comment (Dartemiss) [inline Engine/ResourceManagement/Importer/ModelImporter.cpp:122]: Hola.txt kill it

File: Engine/ResourceManagement/Importer/PrefabImporter.cpp [hunk 2]
```diff
@@ -108,6 +109,11 @@ void PrefabImporter::ExtractGameObjectFromNode
 	node_config.GetString("Name", node_game_object->name, "");
 	node_game_object->original_UUID = node_game_object->UUID;
 
+	if (node_game_object->name == "Level_1_Zone_1_Leaves Tree001_0")
+	{
+		int x = 0;
+	}
+
```
Comment (Dartemiss) [inline Engine/ResourceManagement/Importer/PrefabImporter.cpp:116]: Delete debug stuff

**Review by Dartemiss — APPROVED** [review]
All good mi pana

**Review by yalania — APPROVED** [review]
Test with mushdoom with good scale and also test the animation on the platform and if everything works I approve

**Pull request merged**